docs/sink_config_options.rst

```diff
@@ -0,0 +1,150 @@
+.. _sink-config-options:
+
+JDBC Sink Configuration Options
+-------------------------------
+
+Connection
+^^^^^^^^^^
+
+``connection.url``
+  JDBC connection URL.
+
+  * Type: string
+  * Default: ""
+  * Importance: high
+
+``connection.user``
+  JDBC connection user.
+
+  * Type: string
+  * Importance: high
+
+``connection.password``
+  JDBC connection password.
+
+  * Type: password
+  * Importance: high
+
+Writes
+^^^^^^
+
+``table.name.format``
+  A format string for the destination table name, which may contain '${topic}' as a placeholder for the originating topic name.
+
+  For example, ``kafka_${topic}`` for the topic 'orders' will map to the table name 'kafka_orders'.
+
+  * Type: string
+  * Default: "${topic}"
+  * Importance: medium
+
+``batch.size``
+  Specifies how many records to attempt to batch together for insertion, when possible.
+
+  This config is overridable at the table-level by using a ``$table.`` prefix.
+
+  * Type: int
+  * Default: 3000
+  * Importance: high
+
+
+``insert.mode``
+  The insertion mode to use. Supported modes are:
+
+  `insert`
+
+      Use standard SQL ``INSERT`` statements.
+
+  `upsert`
+
+      Use the appropriate upsert semantics for the target database if it is supported by the connector, e.g. ``INSERT OR IGNORE``.
+
+  This config is overridable at the table-level by using a ``$table.`` prefix.
+
+  * Type: string
+  * Default: "insert"
+  * Importance: medium
+
+.. _sink-pk-config-options:
```

Comment:
Hmm, we may need to figure out how to get these references auto-generated. Otherwise we run the risk of accidentally losing them the next time someone updates this file.